.github/workflows/node.js.yml

```
@@ -16,12 +16,12 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [18.x]
```

```suggestion
        node-version: [16.x, 18.x]
```
temps que 16 est encore dans package.json